apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -435,6 +458,7 @@ export const EventTeamTab = ({
             setAssignAllTeamMembers={setAssignAllTeamMembers}
             teamMembers={teamMembersOptions}
           />
+          {isRoundRobinEventType && <RescheduleRoundRobinHost />}
```

Comment:
Can you move this inside the Advanced tab instead? We should never put anything under the list of hosts, because this list can be long

Reply:
Thanks for your review, I will fix that